tests/scripts/thread-cert/Cert_6_3_01_OrphanReattach.py

```diff
@@ -82,6 +88,31 @@ def test(self):
         for addr in addrs:
             self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER = pv.vars['ROUTER']
+        ED = pv.vars['ED']
+        _epkts = pkts.filter_wpan_src64(ED)
+
+        _epkts.filter_mle_cmd(MLE_CHILD_ID_REQUEST).must_next()
+
+        # Step 2: Remove Router from the network
+        # Step 3: The DUT MUST send three MLE Child Update Requests to its parent
+        for i in range(1, 3):
+            _epkts.filter_mle_cmd(MLE_CHILD_UPDATE_REQUEST).filter_wpan_dst64(ROUTER).must_next().must_verify(
+                lambda p: {SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, MODE_TLV} <= set(p.mle.tlv.type))
+
+        # Step 5: The DUT MUST perform the attach procedure with the Leader
+        _epkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        _epkts.filter_mle_cmd(MLE_CHILD_ID_REQUEST).must_next()
```

Comment:
also verify parent's extended address?

I think we can have a general routine for such commonly used verification:
```suggestion
        # Step 5: The DUT MUST perform the attach procedure with the Leader
        self.verify_attached(pkts, ED, PARENT)
```

Thoughts?

Reply:
agree